examples/meteringlab/c1/cluster.yaml

```diff
@@ -0,0 +1,94 @@
+# Namespace where the Scylla Cluster will be created
```

Comment:
to better imitate serverless env for every scylla cluster you will also need to spawn ScyllaDBMonitoring, example is here: https://github.com/scylladb/scylla-operator/blob/master/examples/monitoring/v1alpha1/scylladbmonitoring.yaml